event-types.md

```diff
@@ -1,12 +1,19 @@
+<!-- START_METADATA
 ---
-title: Events
 sidebar_position: 2
+pagination_next: null
+pagination_prev: null
 ---
+END_METADATA -->
```

Comment:
It should be:
```
title: Events
title: Webhooks API Events
sidebar_label: Events
sidebar_position: 2
description: Webhooks API Events
pagination_next: null
pagination_prev: null
```

Reply:
Sure. Will update. I assume this leads to replacing event type with event too?